php.php

```diff
@@ -0,0 +1,18 @@
+<?php
+    class Deposit {
+
+        /**
+         * The deposit ownner.
+         *
+         * @var string witout info
+         */
+        private $_owner;
+
+        /*
```

Comment:
Typo inside comment in token 'Multiline'. Possible corrections:multiline (13),ultiline (0)